src/main/webapp/WEB-INF/views/admin/project/identification.jsp

```diff
@@ -334,8 +334,8 @@
 	                        <input type="button" value="OSS bulk registration" onclick="fn_grid_com.ossBulkReg('${project.prjId}','11')" class="btnColor red" style="width: 145px;" />
                     	</c:if>
                     	<c:if test="${project.dropYn ne 'Y' and (ct:isAdmin() or project.viewOnlyFlag eq 'N')}">
-                    		<input type="button" value="Check OSS Name" onclick="com_fn.CheckOssViewPage('SRC')" class="btnColor red srcBtn" style="width: 115px;" />
-                    		<input type="button" value="Check License" onclick="com_fn.CheckOssLicenseViewPage('SRC')" class="btnColor red srcBtn" style="width: 100px;" />
+                    		<input type="button" value="Check OSS Name" onclick="com_fn.CheckOssViewPage('SRC')" class="btnColor red srcBtn checkOssName" style="width: 115px;" />
```

Comment:
checkOssName seems unnecessary.

Reply:
Ok, I just removed the unnecessary classes